account_invoice_change_currency/i18n/es.po

```diff
@@ -0,0 +1,73 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* account_invoice_change_currency
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```

Comment:
Don't worry about this, all these files will be re-generated by Weblate automatically from the template. Only change template

Reply:
Done!